keras_cv/layers/preprocessing/with_mixed_precision_test.py

```diff
@@ -0,0 +1,136 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import tensorflow as tf
+from absl.testing import parameterized
+
+from keras_cv.layers import preprocessing
+
+TEST_CONFIGURATIONS = [
+    ("AutoContrast", preprocessing.AutoContrast, {"value_range": (0, 255)}),
+    ("ChannelShuffle", preprocessing.ChannelShuffle, {}),
+    ("Equalization", preprocessing.Equalization, {"value_range": (0, 255)}),
+    (
+        "RandomCropAndResize",
+        preprocessing.RandomCropAndResize,
+        {
+            "target_size": (224, 224),
+            "crop_area_factor": (0.8, 1.0),
+            "aspect_ratio_factor": (3 / 4, 4 / 3),
+        },
+    ),
+    (
+        "RandomlyZoomedCrop",
+        preprocessing.RandomlyZoomedCrop,
+        {
+            "height": 224,
+            "width": 224,
+            "zoom_factor": (0.8, 1.0),
+            "aspect_ratio_factor": (3 / 4, 4 / 3),
+        },
+    ),
+    ("Grayscale", preprocessing.Grayscale, {}),
+    ("GridMask", preprocessing.GridMask, {}),
+    (
+        "Posterization",
+        preprocessing.Posterization,
+        {"bits": 3, "value_range": (0, 255)},
+    ),
+    (
+        "RandomColorDegeneration",
+        preprocessing.RandomColorDegeneration,
+        {"factor": 0.5},
+    ),
+    (
+        "RandomCutout",
+        preprocessing.RandomCutout,
+        {"height_factor": 0.2, "width_factor": 0.2},
+    ),
+    (
+        "RandomHue",
+        preprocessing.RandomHue,
+        {"factor": 0.5, "value_range": (0, 255)},
+    ),
+    (
+        "RandomChannelShift",
+        preprocessing.RandomChannelShift,
+        {"value_range": (0, 255), "factor": 0.5},
+    ),
+    (
+        "RandomColorJitter",
+        preprocessing.RandomColorJitter,
+        {
+            "value_range": (0, 255),
+            "brightness_factor": (-0.2, 0.5),
+            "contrast_factor": (0.5, 0.9),
+            "saturation_factor": (0.5, 0.9),
+            "hue_factor": (0.5, 0.9),
+            "seed": 1,
+        },
+    ),
+    (
+        "RandomGaussianBlur",
+        preprocessing.RandomGaussianBlur,
+        {"kernel_size": 3, "factor": (0.0, 3.0)},
+    ),
+    ("RandomJpegQuality", preprocessing.RandomJpegQuality, {"factor": (75, 100)}),
+    ("RandomSaturation", preprocessing.RandomSaturation, {"factor": 0.5}),
+    (
+        "RandomSharpness",
+        preprocessing.RandomSharpness,
+        {"factor": 0.5, "value_range": (0, 255)},
+    ),
+    ("RandomShear", preprocessing.RandomShear, {"x_factor": 0.3, "x_factor": 0.3}),
+    ("Solarization", preprocessing.Solarization, {"value_range": (0, 255)}),
+]
+
+NO_CPU_FP16_KERNEL_LAYERS = [
+    preprocessing.RandomSaturation,
+    preprocessing.RandomColorJitter,
+]
+
+
+class WithMixedPrecisionTest(tf.test.TestCase, parameterized.TestCase):
+    @parameterized.named_parameters(
+        *TEST_CONFIGURATIONS,
+        ("CutMix", preprocessing.CutMix, {}),
```

Comment:
I think these can just be included in TEST_CONFIGURATIONS

Reply:
Added.